core/src/ibc/client_02/client.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Kodebox, Inc.
```

Comment:
Please split commits.
Since IBC code is using the consensus/lightclient's verification logic, IBC code and consensus/lightclient's code should be included in different commits.